microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-106.md

```diff
@@ -0,0 +1,179 @@
+<!--
+Screenshot:
+1. Install the extension.
+2. Open a file with problems.
+3. Hover over the issue till the lightbulb appears.
+4. Click the light bulb for options.
+-->
+
+<!-- see att. -->
+
+See also:
+* [Automated Quick Fixes and issue filtering](../../../../visual-studio-code/microsoft-edge-devtools-extension.md#automated-quick-fixes-and-issue-filtering) in _Microsoft Edge DevTools extension for Visual Studio Code_
+
+
+<!-- ====================================================================== -->
+## DevTools now uses less disk space when caching sourcemaps
+
+<!-- Subtitle: In Microsoft Edge 106, sourcemaps are now removed from IndexedDB storage if they haven't been accessed in 30 days. -->
+
+Starting in [Microsoft Edge 101](../../../whats-new/2022/04/devtools-101.md#source-maps-are-now-cached-with-indexeddb), DevTools began caching sourcemaps in IndexedDB to reduce the amount of network requests fetching your sourcemaps.  However, this change could potentially take up a lot of disk space if these sourcemaps are never removed.
+
+In Microsoft Edge 106, this issue has been fixed, by removing any sourcemaps that haven't been accessed in 30 days from IndexedDB storage.  Additionally, sourcemaps served from `localhost` are also no longer cached.  Thank you for sending us feedback about this issue!
+
+To understand how DevTools fetches and caches your sourcemaps, watch the **Load status** column in the [Source Maps Monitor tool](../../../source-maps-monitor/source-maps-monitor-tool.md).  When you first load the webpage, the **Source Maps Monitor** tool shows that the source maps aren't cached yet:
+
+![Source Maps Monitor tool, upon first load](./devtools-106-images/source-maps-indexeddb-first-load.png)
+
+When you refresh the webpage, the **Source Maps Monitor** tool shows that the cached source maps are used:
+
+![Source Maps Monitor tool, upon second load](./devtools-106-images/source-maps-indexeddb-second-load.png)
+
+To clear the IndexedDB cache and force DevTools to re-load sourcemaps, run the **Clear source maps cache** command in the Command Menu.  See [Run commands in the Command Menu](../../../command-menu/index.md).
+
+For history, see [Issue 89](https://github.com/MicrosoftEdge/DevTools/issues/89).
+
+See also:
+* [Source maps are now cached with IndexedDB](../../2022/04/devtools-101.md#source-maps-are-now-cached-with-indexeddb) in _What's New in DevTools (Microsoft Edge 101)_
+
+
+<!-- ====================================================================== -->
+## Chromium color themes have been deprecated
+
+<!-- Subtitle: Users of Chromium themes in DevTools will be automatically migrated to the default dark or light theme for a more reliable experience. -->
+
+To improve reliability, the Chromium Light and Chromium Dark themes have been deprecated.  Users who previously had these themes selected will be automatically migrated to the **Light+** or **Dark+** theme, respectively:
```

Comment:
If theme names are going to be bolded, I'd recommend bolding the deprecated theme names as well. No strong opinion either way, just be consistent.

```suggestion
To improve reliability, the **Chromium Light** and **Chromium Dark** themes have been deprecated.  Users who previously had these themes selected will be automatically migrated to the **Light+** or **Dark+** theme, respectively:
```

Reply:
Bolded.  The present presentation includes a UI control; a dropdown list containing UI strings/labels, so pretty easy to make the case for bolding.